src/theory/quantifiers/ceg_t_instantiator.cpp

```diff
@@ -1546,7 +1512,7 @@ void BvInstantiatorPreprocess::process(
 
       if (cur.getKind() == BITVECTOR_EXTRACT)
```

Comment:
What if `cur` is an extract on an arbitrary term? Do we still want to eliminate it or do we only want to eliminate the extracts on variables?

Reply:
We want to eliminate extracts on arbitrary terms